### DISCUSSION TITLE
When using <Wrap>, I'm getting extra space being rendered outside of <html>

### DISCUSSION THREAD

**Discussion by tamsanh** [chakra-ui/chakra-ui#4181] [Q&A] [answered]
Hello,

Sorry, this feels like it shouldn't be an issue, but I'm definitely having difficulty when using the `<Wrap>` element.

For some reason, when I have spacing added to `<Wrap>`, I get extra margin rendered. This especially causes issues when I have a navigation bar, and the page can be scrolled past it.

The following code is the minimum example and attached is a screenshot.

Could I get any hints for this? Perhaps I'm totally in the wrong? Even the spacing example from the documentation gives me the same issue, also attached. Only when spacing is set to 0 does the frame render correctly. Of course, that's less than ideal.

![image](https://user-images.githubusercontent.com/1330789/121017627-125ad600-c7d0-11eb-91cd-233dc16e0448.png)

![image](https://user-images.githubusercontent.com/1330789/121017858-5352ea80-c7d0-11eb-8b36-3faa7ef244a7.png)

```
import {ChakraProvider, Wrap, WrapItem, Center} from '@chakra-ui/react'

function App() {
    return (
        <ChakraProvider>
            <Wrap h={"100vh"} w={"100vw"} spacing={"200px"}>
                <WrapItem>
                    <Center h={"200px"} w={"200px"} shadow={"lg"}>
                        Hello
                    </Center>
                </WrapItem>
                <WrapItem>
                    <Center h={"200px"} w={"200px"} shadow={"lg"}>
                        Hello
                    </Center>
                </WrapItem>
                <WrapItem>
                    <Center h={"200px"} w={"200px"} shadow={"lg"}>
                        Hello
                    </Center>
                </WrapItem>
            </Wrap>
        </ChakraProvider>
    );
}

export default App;
```

![image](https://user-images.githubusercontent.com/1330789/121017163-8b0d6280-c7cf-11eb-9c70-1f11c8708825.png)

**Comment by tamsanh** [comment, marked as answer]
For now, I've added a padding around the Wrap element that is equal to the size of the gap, which seems to work alright.

```
      <Wrap w={"100vw"} h={"100vh"} padding={"20px"} spacing={"20px"}>
        <WrapItem>
          <Center w={"200px"} h={"200px"} shadow={"lg"}>
            Hello
          </Center>
        </WrapItem>
      </Wrap>
```

**Reply by anubra266** [reply]
You're hardcoding the padding,it won't work, if you change the width of the WrapItems

**Reply by tamsanh** [reply]
@anubra266 Oh! What should I do, then?
